package.json

```diff
@@ -152,7 +152,7 @@
     "terser-webpack-plugin": "^5.1.1",
     "ts-jest": "^27.1.5",
     "ts-loader": "^8.0.12",
-    "typescript": "4.1.3",
+    "typescript": "^4.4.4",
```

Comment:
version is still 4.9.5 in the lockfile, you'll need to remove the `^`, install, and then add it back to make sure the lockfile updates

Reply:
fixed it